lib/graphql/schema.rb

```diff
@@ -42,7 +46,7 @@ def initialize(query:, mutation: nil, subscription: nil, max_depth: nil, types:
       @rescue_middleware = GraphQL::Schema::RescueMiddleware.new
       @middleware = [@rescue_middleware]
       # Default to the built-in execution strategy:
-      self.query_execution_strategy = GraphQL::Query::SerialExecution
+      self.query_execution_strategy = GraphQL::Execution::DeferredExecution
```

Comment:
I don't actually want to make this default yet, I just need a way to run the _whole_ test suite on it